virtcontainers/pkg/oci/utils.go

```diff
@@ -323,14 +323,14 @@ func containerCapabilities(s CompatOCISpec) (types.LinuxCapabilities, error) {
 }
 
 // ContainerCapabilities return a LinuxCapabilities for virtcontainer
-func ContainerCapabilities(s CompatOCISpec) (types.LinuxCapabilities, error) {
+func ContainerCapabilities(s CompatOCISpec) (specs.LinuxCapabilities, error) {
```

Comment:
I think this (public) function could be replaced entirely with the existing private `containerCapabilities()` one (just add the `s.Process` check to it). That would confine the use of `CompatOCISpec` to `utils.go` entirely.

Reply:
That sounds great. Hmm, why I didn't think of it....Will modify.